### PR TITLE
external/stdk: Add missing original license

### PR DESCRIPTION
**Pull request by zhouxinhe**
Open Source Validation

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f036cdc2fc4e58645321dbead47df929ba737ceb] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f036cdc2fc4e58645321dbead47df929ba737ceb] - Code Rule Check (C++) OK.

File: external/stdk/tizenrt-wrapper/os/event_groups.h [hunk 1]
```diff
@@ -15,6 +15,33 @@
  * limitations under the License.
  *
  ******************************************************************/
+/*
```
Comment (SRCN-LiQin) [inline external/stdk/tizenrt-wrapper/os/event_groups.h]: how about keep consistent with the license in  event_groups.c ?

Comment (zhouxinhe) [reply external/stdk/tizenrt-wrapper/os/event_groups.h]: OSV tools gives the code matches with event_groups.h in freertos, so I copy the license information from it. 
Actually, event_groups.c in freertos has the same license, but not the BSD-3 license in out event_groups.c.

**Comment by zhouxinhe** [comment]
@sunghan-chang 
Could you please help to review this PR?

File: external/stdk/tizenrt-wrapper/os/event_groups.h (hunk 1 above)
Comment (xixidodo) [reply external/stdk/tizenrt-wrapper/os/event_groups.h]: let's stick with OSV tools.
@sunghan-chang   let's merge this missing license

**Pull request merged**